src/context/sessionContext/index.tsx

```diff
@@ -119,18 +125,19 @@ export const SessionProvider = ({
       url: window.location.href,
       restorePreviousSession: restoreSession,
     })
-      .then((sessionInfo) =>
+      .then(async (sessionInfo) => {
+        if (skipLoadingProfile === true) {
+          return;
+        }
+
         // If handleIncomingRedirect logged the session in, we know what the current
         // user's WebID is.
-        sessionInfo?.webId !== undefined
-          ? getProfileAll(sessionInfo?.webId, {
-              fetch: session.fetch,
-            })
-          : undefined
-      )
```

Comment:
I assume the issue is that `getProfileAll` fails due to a network error if the WebID document isn't provisioned. If that's correct, could this be fixed by catching exceptions, and behaving as if the profile isn't available? If that was possible, it would be easier than having to manually skip this step.

Reply:
I think for now, manually skipping is the best approach, as I don't think we should be pre-fetching the profile/webid anymore, and instead provide a dedicated hook for it.

Reply:
As an aside, due to how this code is structured, the catch at the end can catch more than what it intends to, we should probably switch this from a promise chain to async/await